# Goal
geometry_msgs/PoseStamped target_pose
---
# Result
bool success
---
# Feedback
geometry_msgs/PoseStamped current_pose
